src/Records/MX.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Spatie\Dns\Records;
+
+/**
```

Comment:
Why do we have these docblocks on this class and similar classes? The methods seem to exists on the classes.

Reply:
No, the property exists but the method is called by `__call()` on the `Record` base class.